cm2350/e200z7.py

```diff
@@ -121,6 +121,9 @@ def __init__(self, vw):
         # module registry
         self.modules = {}
 
+        # BitFieldSPR registry
+        self.sprs = {}
+
```

Comment:
if we use the RegisterContext storage created for SPRs (see previous comment about `getRegister()` and `REG_OFFSET_SPR`), I believe this is unnecessary?

Reply:
nm.  Perhaps for future us